packages/block-editor/src/hooks/border.js

```diff
@@ -19,7 +19,6 @@ import { BorderWidthEdit } from './border-width';
 export const BORDER_SUPPORT_KEY = '__experimentalBorder';
 
 export function BorderPanel( props ) {
-	const isDisabled = useIsBorderDisabled( props );
 	const isSupported = hasBorderSupport( props.name );
```

Comment:
I think this can also be cleaned up as it's covered by the individual checks below.
```suggestion
```